public/src/js/models/config/front.js

```diff
@@ -221,9 +222,44 @@ export default class ConfigFront extends BaseClass {
             alert('You must choose a group');
             return;
         }
+        if (this.hasDuplicatePlatformSpecificCollection()) {
+            alert('Cannot share platform-specific collections with other fronts');
+            return;
+        }
         return persistence.front.update(this);
     }
 
+    hasDuplicatePlatformSpecificCollection() {
+        const dups = this.collections.items().filter(c => isPlatformSpecificCollection(c.meta.platform()) && c.parents().length > 1);
+        return dups.length > 0;
+    }
+
+    isUniqueCollection(name) {
+        //Checks if this is the only collection in this front with this name
+        const others = this.collections.items().filter(c => c.meta.displayName() === name);
+        return others.length < 2;
```

Comment:
I thought displayName() always included the platform specific information?

Reply:
`meta.displayName` is the name in the actual collection data (stored in s3). We render it with the platform type in the UI, but do not add it in s3

Reply:
👍 thanks